models/event.js

```diff
@@ -0,0 +1,149 @@
+const express = require('express');
+const app = express();
+const bodyParser = require('body-parser');
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.json());
+
+const MongoClient = require('mongodb').MongoClient,
+	assert = require('assert'),
+	ObjectId = require('mongodb').ObjectID;
+
+const url = 'mongodb://localhost:27017/events';
+const mongoose = require('mongoose');
+mongoose.connect(url);
+mongoose.Promise = require('bluebird');
+module.exports.db = mongoose.connection;
+
+const eventSchema = mongoose.Schema({
+	title: String,
+	description: String,
+	date: String
+}, {collection: "documents"});
+
+Event = mongoose.model('Event', eventSchema);
+
+
+module.exports.insertEvent = function(req, res, callback) {
```

Comment:
All these functions (insertEvent, newEvent, removeEvent, etc.) are part of the controller, not the model. They should go in the controller file where you have the routes defined.